src/index.html

```diff
@@ -20,8 +21,476 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- endregion -->
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="main_page"
+        class="header__logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav header__nav">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li>
```